stop creating jit/generated/ directory

dagitses opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75790
* #75800
* #75699
* #75869
* #75868
* #75698
* #75332
* #75942
* #74849
* #74837
* __->__ #74836

I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

Commit:
stop creating jit/generated/ directory

I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

malfet commented on tools/setup_helpers/generate_code.py:
```diff
@@ -46,8 +46,7 @@ def generate_code(ninja_global: Optional[str] = None,
-    jit_gen_dir = os.path.join(install_dir, 'jit', 'generated')
-    for d in (autograd_gen_dir, jit_gen_dir, python_install_dir):
+    for d in (autograd_gen_dir, python_install_dir):
         if not os.path.exists(d):
             os.makedirs(d)
```
Mind fixing this one as well? (As this code lives in pyton-3.7+ world)
```suggestion
        os.makedirs(d, exist_ok=True)
```

dagitses replied:
fixed!

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]

Commit:
Update on "stop creating jit/generated/ directory"


I don't believe this is used any more.

Differential Revision: [D35187900](https://our.internmc.facebook.com/intern/diff/D35187900/)

[ghstack-poisoned]
